grakn-engine/src/main/java/ai/grakn/engine/HttpHandler.java

```diff
@@ -0,0 +1,255 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.engine;
+
+
+import ai.grakn.engine.controller.AuthController;
+import ai.grakn.engine.controller.CommitLogController;
+import ai.grakn.engine.controller.ConceptController;
+import ai.grakn.engine.controller.DashboardController;
+import ai.grakn.engine.controller.GraqlController;
+import ai.grakn.engine.controller.SystemController;
+import ai.grakn.engine.controller.TasksController;
+import ai.grakn.engine.controller.UserController;
+import ai.grakn.engine.controller.api.AttributeController;
+import ai.grakn.engine.controller.api.AttributeTypeController;
+import ai.grakn.engine.controller.api.EntityController;
+import ai.grakn.engine.controller.api.EntityTypeController;
+import ai.grakn.engine.controller.api.RelationshipController;
+import ai.grakn.engine.controller.api.RelationshipTypeController;
+import ai.grakn.engine.controller.api.RoleController;
+import ai.grakn.engine.controller.api.RuleController;
+import ai.grakn.engine.factory.EngineGraknTxFactory;
+import ai.grakn.engine.session.RemoteSession;
+import ai.grakn.engine.tasks.manager.TaskManager;
+import ai.grakn.engine.user.UsersHandler;
+import ai.grakn.engine.util.JWTHandler;
+import ai.grakn.exception.GraknBackendException;
+import ai.grakn.exception.GraknServerException;
+import ai.grakn.util.REST;
+import com.codahale.metrics.MetricRegistry;
+import mjson.Json;
+import org.apache.http.entity.ContentType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.HaltException;
+import spark.Request;
+import spark.Response;
+import spark.Service;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ExecutorService;
+
+import static ai.grakn.engine.GraknEngineConfig.WEBSOCKET_TIMEOUT;
+
+/**
+ *
+ * @author Michele Orsi
+ */
+public class HttpHandler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GraknEngineServer.class);
```

Comment:
Should we change the logger name to be `getLogger(HttpHandler.class)` for consistency?

In our codebase, every logger is named after the class it belongs to

Reply:
Sure! I maintained the GraknEngineServer name so that my refactor was invisible from a user point of view.

But it makes sense to be consistent with the codebase! I will do fix it!